Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:18.04
+RUN apt-get update \
+    # Install gcc for x86 and ARM
+    && apt-get install -y build-essential gcc-arm-linux-gnueabi gcc-arm-linux-gnueabihf git \
```

Comment:
This should be a new `RUN` command. 

You want to have multiple `RUN` commands so that Docker can store different layers to the image and use cached versions if possible.

Reply:
I did have a multiple RUN since it's easier to debug the script, but according to [this](https://docs.docker.com/develop/dev-best-practices/), they say it's better to have a one line RUN. I was confused by the documents since the immediate layers are removed anyway.

Reply:
solved